fiftyone/utils/torch.py

```diff
@@ -251,32 +252,50 @@ class TorchEmbeddingsMixin(fom.EmbeddingsMixin):
         layer_name (None): the name of the embeddings layer whose output to
             save, or ``None`` if this model instance should not expose
             embeddings. Prepend ``"<"`` to save the input tensor instead
+        feature_extractor (False): whether this model instance should
+            operate as a feature extractor, in which case ``layer_name`` if provided
+            is used to create a feature extractor. If ``layer_name`` is not provided,
+            the model is used as-is for feature extraction.
     """
 
-    def __init__(self, model, layer_name=None):
-        if layer_name is not None:
+    def __init__(self, model, layer_name=None, feature_extractor=False):
+        if feature_extractor:
+            if layer_name:
+                # create a torchvision feature extractor
+                self._model = create_feature_extractor(
```

Comment:
can this be used with all torchmodels or just dino?

Reply:
with all torch models